charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -11,6 +11,7 @@ strategy:
     maxUnavailable: 1
   type: RollingUpdate
 
+
```

Comment:
Are you adding these blank lines intentionally? I made a suggestion above to remove them, but if you're adding them on purpose then disregard the suggestions.
```suggestion
```